scripts/import/import_all_sv_data.pl

```diff
@@ -1854,7 +1850,24 @@ sub post_processing_clinical_significance {
       sv.clinical_significance is not null
     GROUP BY sva.structural_variation_id;
   };
-  $dbVar->do($stmt);
+  my $sth_stmt = $dbVar->prepare($stmt);
+  $sth_stmt->execute();
+  my $all_clin_sign = $sth_stmt->fetchall_arrayref();
+
+  foreach my $data (@{$all_clin_sign}){
+    my $sv_id = $data->[0];
+    my $clin_sign_dup = $data->[1];
+    my @clin_sign_list = split ',', $clin_sign_dup;
+    my %unique_clin_sign;
+    foreach my $key (@clin_sign_list) {
+      $unique_clin_sign{$key} = 1;
+    }
+    my @new_clin_sign_unique = keys %unique_clin_sign;
+    my $clin_sign_unique_final = join ',', @new_clin_sign_unique;
+
+    my $sth_insert_temp = $dbVar->prepare(qq{ INSERT INTO $temp_clin_table (sv_id, clin_sign) values (?, ?) });
```

Comment:
The $sth_insert_temp can be outside of the loop. It doesn't need to newly initialised every time, but can be reused. Just add it before line 1857